src/garage/tf/models/gaussian_cnn_model.py

```diff
@@ -54,13 +52,11 @@ class GaussianCNNModel(Model):
             parameter.
         std_share_network (bool): Boolean for whether mean and std share
             the same network.
-        std_filter_dims(tuple[int]): Dimension of the filters. For example,
-            (3, 5) means there are two convolutional layers. The filter
-            for first layer is of dimension (3 x 3) and the second one is of
-            dimension (5 x 5).
-        std_num_filters(tuple[int]): Number of filters. For example, (3, 32)
-            means there are two convolutional layers. The filter for the first
-            layer has 3 channels and the second one with 32 channels.
+        std_filters (tuple(int, tuple(tuple(int)))): Number and dimension of
```

Comment:
the type uses square brackets and capitals for the types, e.g. 

`Tuple[Tuple[int, Tuple[int, int]], ...]`

More importantly, i think you may have gotten the type spec slightly wrong.

Don't worry, these will be easier to catch when we add a type checker.


see https://docs.python.org/3/library/typing.html for a reference

Reply:
Got it. Thanks